src/core/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon, IconType } from '../icon';
+import { Link } from '../link';
+import { Tag, type ITagProps } from '../tag';
+
+/**
+ * Object representing a sequential segment in the Breadcrumbs path.
+ */
+interface IBreadcrumbsLink {
+    label: string;
+    href?: string;
+}
+
+export interface IBreadcrumbsProps {
+    /**
+     * Array of BreadcrumbsLink objects {label: string, href?: string}
+     * The array indicates depth from the current position to be displayed in the Breadcrumbs.
+     * Starting at index 0 you must define the root up to the current location.
+     * The final index which will render as non-active and without separator.
+     */
+    breadcrumbsOrder: IBreadcrumbsLink[];
```

Comment:
Given that the relative interface is called `IBreadcrumbsLink`, we can call this property simply `links`
```suggestion
    links: IBreadcrumbsLink[];
```